frontend/src/app/main/data/workspace/shapes.cljs

```diff
@@ -319,7 +322,6 @@
            (dc/detach-comment-thread ids)
            (ptk/data-event :layout/update all-parents)
            (dch/commit-changes changes)
-           (ptk/data-event :layout/update layout-ids)
```

Comment:
Why this disappears?

Reply:
The previous `:layout/update` should reflow the layout so the second is not necessary and it's better performance.